spec/restaurant_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'restaurant'
+
+describe Restaurant do
+
+  let(:menu_klass) {double :menu_klass}
+  let(:menu) {double :menu}
+  let(:text_klass) {double :text_klass}
+
+  subject(:restaurant) { described_class.new(menu_klass, text_klass) }
+
+  before do
+    allow(menu_klass).to receive(:new)
+    allow(menu_klass).to receive(:list).and_return :item_1 => 5, :item_2 => 3 
+    allow(text_klass).to receive(:new)
+    allow(text_klass).to receive(:send_message)
+    subject.add_item(:item_1)
+    subject.add_item(:item_2)
+  end
+
+  describe '#display_menu' do
+
+    it 'displays a menu' do
+      expect(subject.display_menu).to eq :item_1 => 5, :item_2 => 3
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)